src/docs/cookbook/networking/web-sockets.md

```diff
@@ -123,25 +125,26 @@ class MyApp extends StatelessWidget {
       title: title,
       home: MyHomePage(
         title: title,
-        channel: IOWebSocketChannel.connect('wss://echo.websocket.org'),
       ),
     );
   }
 }
 
 class MyHomePage extends StatefulWidget {
   final String title;
-  final WebSocketChannel channel;
```

Comment:
I moved the channel directly inside the _MyHomePageState, simplifying a bit the example